src/desktop/apps/experimental_auth_modal/components/App.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
```

Comment:
All new code should be written in TypeScript now that its supported. That said, if this is just an experimental sketch feel free to use whatever!